tidb-cloud/naming-conventions-for-data-import.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Naming Conventions for Data Import
+summary: Learn about the naming conventions of CSV, Parquet, Aurora Snapshot, and SQL files.
+---
+
+# Naming Conventions for Data Import
+
+The Data Import feature in TiDB Cloud supports the following file formats: CSV, Parquet, Aurora Snapshot, and SQL. To make sure that data can be imported successfully, note the following:
+
+- The data files and the schema files for building the database and tables must conform to the naming conventions.
+- If you can not provide the schema file for building the database, TiDB Cloud will create a target database for you according to the default configuration.
+- If you can not provide the schema file for building tables, you need to create the target tables in TiDB Cloud in advance.
+- If you can not provide data files with the required file names, you can modify the file pattern to perform the import task. Otherwise, TiDB Cloud can not scan the files you want to import.
```

Comment:
```suggestion
- Prepare the SQL file for creating the target database (optional) and the SQL file for creating the target table that conform to the naming conventions. If the SQL file for creating the target table is not provided, you need to create the corresponding table manually in the target database in advance.
- Prepare a data file that conforms to the naming conventions for importing data. If the data file name cannot meet the requirements, it is recommended to use the file pattern to perform the import task. Otherwise, the import task can not scan the data files you need to import.
```